docker-compose/min-docker-compose.yml

```diff
@@ -0,0 +1,95 @@
+
+version: '3.2'
+services:
+  data:
+    image: ns1inc/privatedns_data:${TAG:-1.1.0}
+    environment:        # keep environment lines if 2+ containers
+      CONFIG_PORT: 3300 # are on the host machine
+      CONTAINER_NAME: "data"
```

Comment:
this env var `CONTAINER_NAME` doesn't do anything from what I can tell do we even need this?